### PR TITLE
Updated CHANGELOG

### PR DESCRIPTION
**Pull request by utkarsharma2**
# Description
## What is the current behavior?
Update changelog for 1.5.0a1

### CODE REVIEW

File: python-sdk/docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 1.5.0a1
+
+### Feature:
+- Add `load_options` param to accept `pandasOptions` in `load_file` operator [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
```
Comment (kaxil) [inline python-sdk/docs/CHANGELOG.md:6]: Is this `pandasOption`? -- why camel case?

Comment (utkarsharma2) [reply python-sdk/docs/CHANGELOG.md:6]: It's a class name

Comment (sunank200) [reply python-sdk/docs/CHANGELOG.md:6]: `pandasOption` is a class

Comment (utkarsharma2) [reply python-sdk/docs/CHANGELOG.md:6]: ahhh..should be `PandasLoadOptions` will fix in beta/minor release

File: python-sdk/docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,15 @@
...
+- Add `result_format` and `fail_on_empty` params to `run_raw_sql` operator [#1584](https://github.com/astronomer/astro-sdk/pull/1584)
```
Comment (sunank200) [inline python-sdk/docs/CHANGELOG.md:7]: Is there a documentation change for this PR? We should add that as well.

File: python-sdk/docs/CHANGELOG.md (hunk 1 above)
Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:6]: isn't this has been already released in 1.4.0 https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/CHANGELOG.md#feature-1 ?

Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:6]: @kaxil `pandasOption` is a class and we can pass it in `load_options ` so it should be in camel case, WDYT?

Comment (utkarsharma2) [reply python-sdk/docs/CHANGELOG.md:6]: @pankajastro it's added in the milestone for 1.5.0 - https://github.com/astronomer/astro-sdk/milestone/27?closed=1

Comment (kaxil) [reply python-sdk/docs/CHANGELOG.md:6]: @pankajastro it is `PandasOption` :) not `pandasOption`

Comment (kaxil) [reply python-sdk/docs/CHANGELOG.md:6]: We don't use camelCase in Python -- it is popular in Go/Java/JS/C#

Comment (utkarsharma2) [reply python-sdk/docs/CHANGELOG.md:6]: I'll fix this in a minor release/beta release.

Comment (pankajastro) [reply python-sdk/docs/CHANGELOG.md:6]: ohh I typed it wrong :man_facepalming:. The class name is `PandasOption` right? The changelog text should change accordingly

Comment (pankajastro) [inline python-sdk/docs/CHANGELOG.md:6]:
```suggestion
- Add `load_options` param to accept `PandasLoadOptions` in `load_file` operator [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
```

**Pull request closed without merging**